receipt.go

```diff
@@ -29,8 +29,9 @@ func (cli *Client) handleReceipt(node *waBinary.Node) {
 				}
 			}()
 		}
-		cli.dispatchEvent(receipt)
+		go cli.dispatchEvent(receipt)
 	}
+	go cli.sendAck(node)
```

Comment:
`maybeDeferredAck` already handles sending acks. `dispatchEvent` is also intentionally synchronous to allow library users to determine whether they want to handle it in the background or not.